.github/workflows/e2etest.yml

```diff
@@ -148,7 +152,10 @@ jobs:
         run: |
           # Run the ASP.NET Core app as a background job
           cd ./publish/K13
-          Start-Job -ScriptBlock { dotnet ./${{ env.K13_PROJECT_NAME }}.dll } -Name ${{ env.K13_PROJECT_NAME }}
+          Start-Job -ScriptBlock { 
+            dotnet --version
```

Comment:
Does this command have any effect on test execution, or is it just there for debugging?